checker.py

```diff
@@ -40,9 +41,12 @@ def main(files: List[str], draw_cfg: str = ""):
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(
+        description='Static analysis for IEC 61131-3 programs.')
     parser.add_argument("files", nargs='*', help="Path to IEC source files")
     parser.add_argument("--draw-cfg", type=str,
                         help="Save control flow graph image at the selected path")
+    parser.add_argument("-b","--binary", default="../output/bin/iec_checker",
```

Comment:
Maybe we should replace the default value with `output/bin/iec_checker`? The `../output/bin/iec_checker` value is used to run the test suite, so I assume that the checker will be executed from the `tests` directory.

Reply:
I definitely agree! I wanted to keep the content as close to the original as possible, but I think you're right, using `os.path` would definitely be good!